t/050-ttl.t

```diff
@@ -17,13 +17,26 @@ local ttl = 0.1
 
 test:ok(queue, 'queue is loaded')
 
+local function test_take_after_ttl(test, tube, ttl)
```

Comment:
- Can't we just check the result after `ttl + ttl/2`
- Use of skip is invalid if `after - before < ttl * 1000000` we must still wait and recheck the result

Reply:
> I think we can neglect this probability in the test.

I think it wil be better to give a meaningful error.

> * Can't we just check the result after `ttl + ttl/2`

We can, but it depends on what we decide to do with system clock change.

>     * Use of skip is invalid if `after - before < ttl * 1000000` we must still wait and recheck the result

Please see the update. If you don't like it, then I'll just replace `fibe.sleep(ttl)` with `fiber.sleep(ttl + ttl/2)`. It will almost always work and will be easier.